### DISCUSSION TITLE
Expo React Native Firebase IOS Notification Images Multiple Commands Produce Error

### DISCUSSION THREAD

**Discussion by erdincakdoganmoi** [invertase/react-native-firebase#8154] [General]
Hello guys . I m working on Expo Bare Worflow. 
I try to use below documentation for IOS Notification Images
https://rnfirebase.io/messaging/ios-notification-images
But when I used this way I m facing same problem like below

<img width="693" alt="Screenshot 2024-11-21 at 12 27 12" src="https://github.com/user-attachments/assets/af05f8ea-5ec4-4738-b027-eec144ab110b">

  "dependencies": {
    "@expo-google-fonts/inter": "^0.2.3",
    "@expo/vector-icons": "^14.0.2",
    "@gorhom/bottom-sheet": "^4",
    "@hookform/resolvers": "^3.9.0",
    "@react-native-async-storage/async-storage": "^1.23.1",
    "@react-native-community/checkbox": "^0.5.17",
    "@react-native-community/netinfo": "^11.3.2",
    "@react-native-firebase/analytics": "^20.5.0",
    "@react-native-firebase/app": "^20.0.0",
    "@react-native-firebase/crashlytics": "^20.0.0",
    "@react-native-firebase/messaging": "^20.0.0",
    "@react-native-picker/picker": "^2.7.7",
    "@react-navigation/bottom-tabs": "^6.5.20",
    "@react-navigation/native": "^6.1.17",
    "@react-navigation/native-stack": "^6.11.0",
    "@react-navigation/stack": "^6.3.29",
    "@tanstack/react-query": "^5.49.2",
    "axios": "^1.7.2",
    "expo": "~51.0.14",
    "expo-app-loading": "^2.1.1",
    "expo-application": "~5.9.1",
    "expo-blur": "~13.0.2",
    "expo-build-properties": "~0.12.1",
    "expo-checkbox": "~3.0.0",
    "expo-clipboard": "^6.0.3",
    "expo-constants": "^16.0.2",
    "expo-crypto": "~13.0.2",
    "expo-dev-client": "~4.0.18",
    "expo-device": "~6.0.2",
    "expo-font": "~12.0.7",
    "expo-image": "~1.12.12",
    "expo-image-picker": "~15.0.7",
    "expo-linking": "~6.3.1",
    "expo-localization": "~15.0.3",
    "expo-modules-autolinking": "^1.11.1",
    "expo-modules-core": "^1.12.15",
    "expo-notifications": "~0.28.9",
    "expo-splash-screen": "~0.27.5",
    "expo-status-bar": "~1.12.1",
    "expo-updates": "^0.25.20",
    "i18n-js": "^4.4.3",
    "i18next": "^23.15.1",
    "ky": "^1.4.0",
    "lottie-react-native": "6",
    "moment": "^2.30.1",
    "react": "18.2.0",
    "react-hook-form": "^7.52.1",
    "react-i18next": "^15.0.2",
    "react-native": "0.74.2",
    "react-native-chart-kit": "^6.12.0",
    "react-native-code-input": "^1.0.6",
    "react-native-confirmation-code-field": "^7.4.0",
    "react-native-country-code-picker": "^1.3.5",
    "react-native-country-picker-modal": "^2.0.0",
    "react-native-element-dropdown": "^2.12.1",
    "react-native-gesture-handler": "^2.17.1",
    "react-native-linear-gradient": "^2.8.3",
    "react-native-notificated": "^0.1.6",
    "react-native-phone-input": "^1.3.7",
    "react-native-picker-select": "^9.2.0",
    "react-native-reanimated": "^3.14.0",
    "react-native-safe-area-context": "4.10.1",
    "react-native-screens": "^3.31.1",
    "react-native-svg": "^15.3.0",
    "react-native-url-polyfill": "^2.0.0",
    "react-query": "^3.39.3",
    "reactotron-react-native": "^5.1.8",
    "zod": "^3.23.8",
    "zustand": "^4.5.4"
  },
  "devDependencies": {
    "@babel/core": "^7.20.0",
    "@eslint/js": "^9.6.0",
    "@types/expo__vector-icons": "^10.0.0",
    "@types/react": "^18.3.3",
    "@types/react-dom": "^18.3.0",
    "@types/react-native": "^0.73.0",
    "eslint": "9.x",
    "eslint-config-expo": "^7.1.2",
    "eslint-config-prettier": "^9.1.0",
    "eslint-plugin-prettier": "^5.1.3",
    "globals": "^15.8.0",
    "prettier": "^3.3.2",
    "react-native-svg-transformer": "^1.4.0",
    "typescript": "^5.1.3",
    "typescript-eslint": "^7.15.0"
  },
  
  Do you any see any error for like me ?
  Thank you for your help

**Comment by mikehardy** [comment]
```

"@react-native-firebase/analytics": "^20.5.0",
"@react-native-firebase/app": "^20.0.0",
"@react-native-firebase/crashlytics": "^20.0.0",
"@react-native-firebase/messaging": "^20.0.0",

"react-native": "0.74.2",

```

My initial reaction is to not look too closely until you've reproduced on current versions - updates are released to fix problems, frequently build issues with new versions of toolchains.

You've also basically logged an issue but done so in the "discussions" area so you're ignoring our issue template which has important information required (like, are you using `use_frameworks! :linkage => :static` ? or in an Expo context using expo-build-properties and setting the ios pod integration to frameworks / static? Hope so, otherwise you'll get build errors

**Reply by erdincakdoganmoi** [reply]
Yes, you are right, it would be more correct to open it there. I am using Expo, but I did it with the method on the React Native Firebase website.

I used the code there completely and this is the error I get when I get Build from Xcode. I hope I was able to explain it.

Based on what you said, would downgrading Firebase version work?

**Reply by mikehardy** [reply]
🤔 
- I was not saying "go log it in expo", I was saying "are you sure you configured your build correctly?" https://rnfirebase.io/#configure-react-native-firebase-modules
- it is not adequately explained, yet
- I said "My initial reaction is to not look too closely until you've reproduced on current versions" - I still think the same, so no, I do not think downgrading would help in any way. Quite the opposite, if you want attention for issues in an open source context, it's considered a barrier to entry that you reproduce your issues on current versions of everything so that no one's limited time is wasted chasing problems that are already fixed

**Reply by erdincakdoganmoi** [reply]
-Firebase Cloud Messaging is installed and running in my project. 
-All I wanted to do was add the IOS Notification Image. If that's what you're asking
